util/searchRankScript.bash

```diff
@@ -48,26 +50,43 @@ fi
 [[ -d "$exampleDir" ]] && score=$((score+1)) && echo "example dir found in package"
 [[ -d "$srcDir" ]] && score=$((score+1)) && echo "src dir found in package"
 
-# points for number of examples 
+# points for number of examples (max 3)
 countForExamples=$(ls -l $exampleDir/*.chpl 2>/dev/null | wc -l)
 echo "Found $countForExamples examples in package."
+maxScoreExamples=3
+if [ "$countForExamples" -gt 3 ] 
```

Comment:
From what I understand, the previous implementation gave 1 point per example. Now it gives 3 points if the package has at least 3 examples. Is this the intended design, or should it give 1 point per example, up to 3 points for 3 examples? Similarly for tests.

Reply:
If leaving this implementation as is, I'd suggest changing:

```suggestion
if [ "$countForExamples" -gt ${maxScoreExamples} ] 
```

to eliminate magic numbers and reduce number of changes required if we decide to change the value of `maxScoreExamples` (same for tests)

Reply:
@ben-albrecht - The previous implementation gave 1 point for each test or example found in their respective directory and the score generated could be any arbitrary number depending on number of tests or examples found. But due to normalization of scores, we decided (in our weekly call) that, if number of examples exceeds 3, then we set the value of score to 3, i.e, the maximum possible score that a package can get for tests & examples is 3.

Reply:
I had misunderstood the code when I wrote my first question. Makes sense now. Thanks!